### DISCUSSION TITLE
Configure script on bare metal server network interfaces all shows unknown link status

### DISCUSSION THREAD

**Discussion by Thor192** [Security-Onion-Solutions/securityonion#13542] [Unsupported Versions]
`Management IP could not be determined`
Are trying to Configure a fresh install of security onion with the latest installer ISO(2.4.90-20240729). I boot in and I go to do the network configuration all the links in the menu show as links that is unknown although the link that is plugged in has light. None of the other interfaces will get a link like when I'm not in the OS, I do get link on all interfaces. I went into the cli and manually set the interface to up that not change the outcome. I can ping the gateway (firewall) it allows ping from trust zone. This server does have a large number of interfaces 8 total if that matters. It can't write anything out to the log because it says it's a read only file system. I'm not sure how to proceed.  Oh I've also tried multiple interfaces on the upstream switch side verified the configuration there. When I bring the interface up manually I do see a Mac address being learned on that port.

**Comment by cm-ops** [comment]
I would investigate this `read only file system` first.

**Reply by Thor192** [reply]
Fair enough I'll look into it. I don't know why it would be coming up that way i'll check my bios settings, Maybe take out the drive format it.
